tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -57,120 +57,130 @@ describe('Acceptance: addon-smoke-test', function() {
+        ? 'kitchen-sink-mu'
+        : 'kitchen-sink';
+      yield copyFixtureFiles(`addon/${fixtureFile}`);
 
-    let packageJsonPath = path.join(addonRoot, 'package.json');
-    let packageJson = fs.readJsonSync(packageJsonPath);
+      let packageJsonPath = path.join(addonRoot, 'package.json');
+      let packageJson = fs.readJsonSync(packageJsonPath);
 
-    expect(packageJson.devDependencies['ember-source']).to.not.be.empty;
+      expect(packageJson.devDependencies['ember-source']).to.not.be.empty;
 
-    packageJson.dependencies = packageJson.dependencies || {};
-    // add HTMLBars for templates (generators do this automatically when components/templates are added)
-    packageJson.dependencies['ember-cli-htmlbars'] = 'latest';
+      packageJson.dependencies = packageJson.dependencies || {};
+      // add HTMLBars for templates (generators do this automatically when components/templates are added)
+      packageJson.dependencies['ember-cli-htmlbars'] = 'latest';
 
-    fs.writeJsonSync(packageJsonPath, packageJson);
+      fs.writeJsonSync(packageJsonPath, packageJson);
 
-    let result = yield runCommand('node_modules/ember-cli/bin/ember', 'build');
+      let result = yield runCommand('node_modules/ember-cli/bin/ember', 'build');
 
-    expect(result.code).to.eql(0);
-    let contents;
+      expect(result.code).to.eql(0);
+      let contents;
 
-    let indexPath = path.join(addonRoot, 'dist', 'index.html');
-    contents = fs.readFileSync(indexPath, { encoding: 'utf8' });
-    expect(contents).to.contain('"SOME AWESOME STUFF"');
+      let indexPath = path.join(addonRoot, 'dist', 'index.html');
+      contents = fs.readFileSync(indexPath, { encoding: 'utf8' });
+      expect(contents).to.contain('"SOME AWESOME STUFF"');
 
-    let cssPath = path.join(addonRoot, 'dist', 'assets', 'vendor.css');
-    contents = fs.readFileSync(cssPath, { encoding: 'utf8' });
-    expect(contents).to.contain('addon/styles/app.css is present');
+      let cssPath = path.join(addonRoot, 'dist', 'assets', 'vendor.css');
+      contents = fs.readFileSync(cssPath, { encoding: 'utf8' });
+      expect(contents).to.contain('addon/styles/app.css is present');
 
-    let robotsPath = path.join(addonRoot, 'dist', 'robots.txt');
-    contents = fs.readFileSync(robotsPath, { encoding: 'utf8' });
-    expect(contents).to.contain('tests/dummy/public/robots.txt is present');
+      let robotsPath = path.join(addonRoot, 'dist', 'robots.txt');
+      contents = fs.readFileSync(robotsPath, { encoding: 'utf8' });
+      expect(contents).to.contain('tests/dummy/public/robots.txt is present');
 
-    result = yield runCommand('node_modules/ember-cli/bin/ember', 'test');
+      result = yield runCommand('node_modules/ember-cli/bin/ember', 'test');
 
-    expect(result.code).to.eql(0);
-  }));
+      expect(result.code).to.eql(0);
+    })
+  );
+
+  it(
+    'npm pack does not include unnecessary files',
+    co.wrap(function*() {
```

Comment:
ouch

Reply:
we can set `printWidth: 120` if that helps?